src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ ov_add_test_target(
             PUBLIC
                 conformance_shared
                 op_conformance_utils
+                sharedTestClasses
```

Comment:
Why this is required?

Reply:
this is duplication, because of the lib is linked via `conformance_shared`

Reply:
fixed